src/pages/public_registration/ContainerLink.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Link } from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
+import LabelContentRow from '../../components/LabelContentRow';
+import { RegistrationFieldName } from '../../types/publicationFieldNames';
+import useSearchRegistrations from '../../utils/hooks/useSearchRegistrations';
+import { getRegistrationLandingPagePath } from '../../utils/urlPaths';
+
+interface ContainerLinkProps {
+  containerId: string;
+  label: string;
+}
+
+const ContainerLink = (props: ContainerLinkProps) => {
```

Comment:
huff. kan vi bruke et mer beskrivende navn? jeg skjønner ikke hva dette er uten å måtte sette med inn i koden 😝

Reply:
noen grunn til at du velger å ikke destrukturere props btw?

Reply:
tjaa, denne komponenten skal egentlig bare vise noen få utvalgte felter (for øyeblikket bare tittel, men blir nok mer ganske snart, feks. `journal`, `series`, `publisher`, etc) for en "parent" Registrering basert på id... Kanskje bedre med `<RegistrationSummary id={...} />`? eller `RegistrationContext`?

Reply:
var en tidligere komponent som hadde ganske mange props at det ble mye ryddigere å bare bruke `props` sånn som her. Men siden det bare er 2 her er det kanskje bedre å destrukturere ja

Reply:
Ja takk til RegistrationSummary. Myyye mer forklarende